CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog].
 
 ## Unreleased
 ### Features
+* Compatibility with evil-mode's jump list and search histoy features
+  was included in [#59]. This integration happens automatically and
+  requires no user input/customisation.
```

Comment:
This seems like it belongs in the Enhancements section rather than Features, although I could be convinced otherwise.

Reply:
done